test/fixtures/url-tests.js

```diff
@@ -5110,6 +5110,246 @@ module.exports =
     "search": "?test",
     "hash": "#x"
   },
+  "# File URLs and many (back)slashes",
```

Comment:
Done synchronising them all by hands, and the lines are added as a result! thanks.